packages/ui/client/components/FileDetails.vue

```diff
@@ -29,6 +29,9 @@ const open = () => {
 const changeViewMode = (view: Params['view']) => {
   viewMode.value = view
 }
+const consoleCount = computed(() => {
+  return currentLogs.value?.map(l => l.size).reduce((s, a) => s + a, 0) ?? 0
```

Comment:
You use map and reduce, why don't do both in reduce?
```suggestion
  return currentLogs.value?.reduce((s, a) => s + a.size, 0) ?? 0
```

Reply:
fixed